.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 366
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 40
```

Comment:
Why 40? 30 or 42 would be nicer :P

Reply:
Because you said "more than one month" :D 42 it is then.